test/database/create_storage_tables_user_attachments_migration.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class CreateStorageTablesUserAttachmentsMigration < ActiveRecord::Migration[7.1]
+  def up
+    create_table :storage_tables_user_avatar_attachments, primary_key: [:blob_key, :record_id, :checksum],
```

Comment:
Het is een dummy, maar denk dat je hier altijd de record_id vooraan wilt omdat daar altijd op gequeried gaat worden.

```suggestion
    create_table :storage_tables_user_avatar_attachments, primary_key: [:record_id, :blob_key, :checksum],
```